Brocfile.js

```diff
@@ -0,0 +1,35 @@
+/* global require, module */
+
+var EmberAddon = require('ember-cli/lib/broccoli/ember-addon');
+var fs = require('fs');
+
+// TODO(azirbel): Use something more legit
```

Comment:
I think JQuery has a method that can do this for you? This is alright as is but you might want to add a rule for single quote (replace `'` with `&#39;`)